### PR TITLE
chore(manifest): add observability field to rdws

### PR DESCRIPTION
**Pull request by Lou1415926**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/manifest/rd_web_svc.go [hunk 1]
```diff
@@ -32,6 +32,15 @@ type RequestDrivenWebServiceConfig struct {
 	Tags                              map[string]string                    `yaml:"tags"`
 	PublishConfig                     PublishConfig                        `yaml:"publish"`
 	Network                           RequestDrivenWebServiceNetworkConfig `yaml:"network"`
+	Observability                     Observability                        `yaml:"observability"`
+}
+
```
Comment (iamhopaul123) [inline internal/pkg/manifest/rd_web_svc.go:37]: Comment pls

File: internal/pkg/manifest/validate.go [hunk 2]
```diff
@@ -49,6 +51,8 @@ var (
 
 	essentialContainerDependsOnValidStatuses = []string{dependsOnStart, dependsOnHealthy}
 	dependsOnValidStatuses                   = []string{dependsOnStart, dependsOnComplete, dependsOnSuccess, dependsOnHealthy}
+	nlbValidProtocols                        = []string{TCP, tls}
```
Comment (iamhopaul123) [inline internal/pkg/manifest/validate.go:54]: Why not `TLS`?

Comment (Lou1415926) [reply internal/pkg/manifest/validate.go:54]: `TCP` is exported while `tls` doesn't need to be.

Comment (iamhopaul123) [reply internal/pkg/manifest/validate.go:54]: ooh sorry i misread it.

**Review by iamhopaul123 — APPROVED** [review]

**Review by huanjani — APPROVED** [review]
LGTM!

**Pull request merged**